dts/arm/renesas/ra/ra8/ra8x1.dtsi

```diff
@@ -0,0 +1,241 @@
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		gpio5: gpio@404000a0 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x404000a0 0x20>;
+			port = <5>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		gpio6: gpio@404000c0 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x404000c0 0x20>;
+			port = <6>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		gpio7: gpio@404000e0 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x404000e0 0x20>;
+			port = <7>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		gpio8: gpio@40400100 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x40400100 0x20>;
+			port = <8>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		gpio9: gpio@40400120 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x40400120 0x20>;
+			port = <9>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		gpioa: gpio@40400140 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x40400140 0x20>;
+			port = <10>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		gpiob: gpio@40400160 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x40400160 0x20>;
+			port = <11>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		sci0: sci0@40358000 {
+			compatible = "renesas,ra8-uart-sci-b";
+			channel = <0>;
+			interrupts = <4 1>, <5 1>, <6 1>, <7 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40358000 0x100>;
+			status = "disabled";
+		};
+
+		sci1: sci1@40358100 {
+			compatible = "renesas,ra8-uart-sci-b";
+			channel = <1>;
+			interrupts = <8 1>, <9 1>, <10 1>, <11 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40358100 0x100>;
+			status = "disabled";
+		};
+
+		sci2: sci2@40358200 {
+			compatible = "renesas,ra8-uart-sci-b";
+			channel = <2>;
+			interrupts = <12 1>, <13 1>, <14 1>, <15 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40358200 0x100>;
+			status = "disabled";
+		};
+
+		sci3: sci3@40358300 {
+			compatible = "renesas,ra8-uart-sci-b";
+			channel = <3>;
+			interrupts = <16 1>, <17 1>, <18 1>, <19 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40358300 0x100>;
+			status = "disabled";
+		};
+
+		sci4: sci4@40358400 {
+			compatible = "renesas,ra8-uart-sci-b";
+			channel = <4>;
+			interrupts = <20 1>, <21 1>, <22 1>, <23 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40358400 0x100>;
+			status = "disabled";
+		};
+
+		sci9: sci9@40358900 {
+			compatible = "renesas,ra8-uart-sci-b";
+			channel = <9>;
+			interrupts = <24 1>, <25 1>, <26 1>, <27 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40358900 0x100>;
+			status = "disabled";
+		};
```

Comment:
@soburi : Thank you for your feedback. Currently, we haven't planned to support I2C, SPI and Smart card interface on the SCI.

Reply:
Since devicetree is a hardware description, it should be written based on hardware configuration, not support status.
Since SCI is a common device with several communication methods to choose from, such a devicetree representation should be given.

NXP's FlexComm has functionality very similar to SCI, but even in that case, functions are split into child nodes, so SCI is also necessary to separate the common parts of shared part and each function.

https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/nxp/nxp_mcxn94x_common.dtsi#L173-L206